var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -110,6 +112,10 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
 
     generator("ninja")
     depends_on("ninja@1.10:", type="build")
+    
+    build_directory = 'spack-build'
```

Comment:
I am pretty sure spack uses out-of-tree builds by default already.
In particular since we check for this in DBCSR and would fail without:
https://github.com/cp2k/dbcsr/blob/6e11798d8e03f5a64fdbdfcacca83465ace118d8/CMakeLists.txt#L7-L14

Reply:
@dev-zero
I've fixed it.